File: pbworkspace/pbworkspace-jar/src/main/java/it/csi/pbandi/pbworkspace/integration/vo/BandoVO.java
/*******************************************************************************
* Copyright Regione Piemonte - 2024
* SPDX-License-Identifier: EUPL-1.2
******************************************************************************/

package it.csi.pbandi.pbworkspace.integration.vo;

public class BandoVO {

	private String nomeBandoLinea;
	private Long processo;
	private Long progrBandoLineaIntervento;

	public String getNomeBandoLinea() {
		return nomeBandoLinea;
	}

	public void setNomeBandoLinea(String nomeBandoLinea) {
		this.nomeBandoLinea = nomeBandoLinea;
	}

	public Long getProcesso() {
		return processo;
	}

	public void setProcesso(Long processo) {
		this.processo = processo;
	}

	public Long getProgrBandoLineaIntervento() {
		return progrBandoLineaIntervento;
	}

	public void setProgrBandoLineaIntervento(Long progrBandoLineaIntervento) {
		this.progrBandoLineaIntervento = progrBandoLineaIntervento;
	}

	@Override
	public String toString() {
		return "BandoVO [nomeBandoLinea=" + nomeBandoLinea + ", processo=" + processo + ", progrBandoLineaIntervento="
				+ progrBandoLineaIntervento + "]";
	}

	

}

File: pbwebbowcl/src/app/configurazione-bando/commons/vo/bando-vo.ts
/*
 * Copyright Regione Piemonte - 2024 
 * SPDX-License-Identifier: EUPL-1.2 
*/

export class BandoVo { 
    constructor(
        public descMateria: string,
        public descModalitaAttivazione: string,
        public idBando: number,
        public idBandoLinea: number,
        public idLineaDiIntervento: number,
        public nomeBandoLinea: string,
        public titoloBando: string
    ) { }
}